src/vs/editor/common/config/editorOptions.ts

```diff
@@ -2734,13 +2734,13 @@ export interface IEditorLightbulbOptions {
 	 * Enable the lightbulb code action.
 	 * Defaults to true.
 	 */
-	enabled?: boolean;
+	enabled?: ShowLightbulbIconMode;
 
 	experimental?: {
 		/**
 		 * Highlight AI code actions with AI icon
 		 */
-		showAiIcon?: ShowAiIconMode;
+		showAiIcon?: boolean;
```

Comment:
Because they don't like the sparkle icon and like to see a light-bulb instead? Then we need to make the sparkle icon it prettier, not add another setting.